src/static/icons/gifting.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="16" height="18" viewBox="0 0 16 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I imagine it doesn't exist, but ideally icons would come from Source.

Reply:
Yeah that'd be ideal, unfortunately it doesn't exist yet and is a bit of a last minute addition.